forge-gui/res/cardsfolder/upcoming/relive_the_past.txt

```diff
@@ -0,0 +1,10 @@
+Name:Relive the Past
+ManaCost:5 G W
+Types:Sorcery
+A:SP$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Artifact.YouOwn | TargetMin$ 0 | TargetMax$ 1 | SubAbility$ DBLand |  TgtPrompt$ Select up to one target artifact from your graveyard | SpellDescription$ Return up to one target artifact card, up to one target land card, and up to one target non-Aura enchantment card from your graveyard to the battlefield. 
+SVar:DBLand:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Land.YouOwn | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target land from your graveyard | SubAbility$ DBEnchantment
+SVar:DBEnchantment:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | SubAbility$ DBAnimate | ValidTgts$ Enchantment.YouOwn+nonAura | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target non-aura enchantment from your graveyard 
+SVar:DBAnimate:DB$ Animate | Defined$ Targeted | Types$ Creature,Elemental | Power$ 5 | Toughness$ 5 | Duration$ Permanent | SpellDescription$ They are 5/5 Elemental creatures in addition to their other types.
```

Comment:
this needs to be reworked into a single ChangeZone and AnimateSubAbility

Reply:
So I guess 3 choosecard lines, changezoneall with AnimateSubAbility ?

Reply:
yea or Pump as alternative
whatever behaves better ingame

Reply:
are you sure you tested it now?

Reply:
It did work during testing yesterday.

Do you prefer to use a remembered instead of targeted ?

Reply:
I claim it doesn't if you target more than one :P

Reply:
Thanks working now 
![Uploading image.png…]()